raiden-dapp/src/views/Transfer.vue

```diff
@@ -26,15 +26,14 @@
             </span>
           </v-tooltip>
         </div>
-        <div
-          class="transfer__token-networks__dropdown"
+        <action-button
+          :text="token.name"
+          ghost
+          enabled
+          full-width
+          class="transfer__top-button"
           @click="showTokenNetworks = true"
-        >
-          <span>{{ token.name }}</span>
-          <span>
-            <down-arrow />
```

Comment:
Please remove component as well if unused

Reply:
Didn't see it wasn't being used anywhere else. Thanks